NEPT-475: Implement behat tests for multisite registration feature

alba-everis opened the pull request:
## NEPT-475

### Description

Implement behat tests for multisite registration feature

### Change log

- Added: Added tests for registration features.


### Commands

No commands needed.

alba-everis committed:
NEPT-475: Add test feature for multisite registrations

alba-everis committed:
NEPT-475: Add feature to test registration in Next Europa Communities

alba-everis committed:
NEPT-475: Clean code in multisite_registration_standard.feature

alba-everis committed:
NEPT-475: Add scenario for testing closed registration

imanoleguskiza committed:
Merge branch 'master' into nept-475

imanoleguskiza commented on tests/features/multisite_registration/multisite_registration_communities.feature:
```diff
@@ -0,0 +1,175 @@
+@api @javascript
+Feature: Multisite registration standard
```
I believe the name of the feature should be "Multisite registration og"

imanoleguskiza commented on tests/features/multisite_registration/multisite_registration_communities.feature:
```diff
@@ -0,0 +1,175 @@
+@api @javascript
```
Since this feature refers to the og (or communities) version of the platform, you should add the tag @communities

imanoleguskiza reviewed (dismissed):
Currently the build is red and I think its because the tests are being run on the incorrect builds, lets fix my comments and see the results

alba-everis committed:
Merge branch 'master' into nept-475

alba-everis committed:
Merge branch 'nept-475' of https://github.com/ec-europa/platform-dev into nept-475

alba-everis committed:
NEPT-475: Change name of feature and add @commutities tag

alba-everis committed:
NEPT-475: Add @theme_wip tags in some scenarios that test blocks not used in ec_europa.

alba-everis committed:
Merge branch 'master' into nept-475

ku-enza committed:
Merge branch 'master' into nept-475

Commit:
nept-475: Add @theme_wip tag on scenario.

Commit:
Merge branch 'nept-475' of https://github.com/ec-europa/platform-dev into nept-475

ku-enza committed:
Merge branch 'master' into nept-475

ku-enza committed:
Merge branch 'master' into nept-475

ku-enza committed:
Merge branch 'master' into nept-475

enriquelacoma committed:
NEPT-2209: multisite_drupal_toolbox_linkit_scan_url error (#2332)

* NEPT-2209: Update function multisite_drupal_toolbox_linkit_scan_url to
work with drupal_parse_url

* NEPT-2209: Remove unused variable

* NEPT-2209: Force the use of protocol on the url.

* NEPT-2209: Check if the url has schema

* NEPT-2209: Bypass phpcs check, we need to use parse_url() function

* NEPT-2209: Update function multisite_drupal_toolbox_linkit_scan_url to
work with drupal_parse_url

* NEPT-2209: Remove unused variable

* NEPT-2209: Force the use of protocol on the url.

* NEPT-2209: Check if the url has schema

* NEPT-2209: Bypass phpcs check, we need to use parse_url() function

enriquelacoma committed:
NEPT-2209: multisite_drupal_toolbox_linkit_scan_url error (#2332)

* NEPT-2209: Update function multisite_drupal_toolbox_linkit_scan_url to
work with drupal_parse_url

* NEPT-2209: Remove unused variable

* NEPT-2209: Force the use of protocol on the url.

* NEPT-2209: Check if the url has schema

* NEPT-2209: Bypass phpcs check, we need to use parse_url() function

* NEPT-2209: Update function multisite_drupal_toolbox_linkit_scan_url to
work with drupal_parse_url

* NEPT-2209: Remove unused variable

* NEPT-2209: Force the use of protocol on the url.

* NEPT-2209: Check if the url has schema

* NEPT-2209: Bypass phpcs check, we need to use parse_url() function

AlbaParrilla committed:
NEPT-2162: Add condition. (#2345)

Ok, nice improvement being backported

enriquelacoma committed:
NEPT-2181: Module to show message when publishing a node after the configured date. (#2311)

* NEPT-2181: Module to show message when scheduling a node publication.

* NEPT-2181: Compare dates using cet timezone

* NEPT-2181: Use cet timezone for date comparation

* NEPT-2181: Fix conding standars

* NEPT-2181: Fix coding standars

*  NEPT-2181: Handle publication date when using unix time.
Update message to include the date to check.

* NEPT-2181: Update messages and handle revisions.

* NEPT-2181: Create scheduled times for publication with current timezone

* NEPT-2181: Use unix time to compare dates

* NEPT-2181: Add symlink on communities profile

* NEPT-2181: Use t() to show message

* NEPT-2181: Remove t()

* NEPT-2181: Allow message translation

* NEPT-2181: Ignore phpcs for t() call with variable

* NEPT-2181: Module to show message when scheduling a node publication.

* NEPT-2181: Compare dates using cet timezone

* NEPT-2181: Use cet timezone for date comparation

* NEPT-2181: Fix conding standars

* NEPT-2181: Fix coding standars

*  NEPT-2181: Handle publication date when using unix time.
Update message to include the date to check.

* NEPT-2181: Update messages and handle revisions.

* NEPT-2181: Create scheduled times for publication with current timezone

* NEPT-2181: Use unix time to compare dates

* NEPT-2181: Add symlink on communities profile

* NEPT-2181: Use t() to show message

* NEPT-2181: Remove t()

* NEPT-2181: Allow message translation

* NEPT-2181: Ignore phpcs for t() call with variable

* NEPT-2181: Show message on the edition form, not on save.

* NEPT-2181: Remove extra line

* NEPT-2181: Show message as warning

Fefaine committed:
NEPT-2220: Instruct phing to delete the xmlsitemap_user (#2343)

* NEPT-2220: Instruct phing to delete the xmlsitemap_user after xmlsitemap module is downloaded.

* NEPT-2220: Remove uninstall step.

Fefaine committed:
NEPT-2220: Instruct phing to delete the xmlsitemap_user (#2343)

* NEPT-2220: Instruct phing to delete the xmlsitemap_user after xmlsitemap module is downloaded.

* NEPT-2220: Remove uninstall step.

enriquelacoma committed:
NEPT-1241: Enable ecas behat test (#1906)

*  NEPT-1241: Enable ecas behat test

* NEPT-1241: Remove todo comment

enriquelacoma committed:
NEPT-1241: Enable ecas behat test (#1906)

*  NEPT-1241: Enable ecas behat test

* NEPT-1241: Remove todo comment

enriquelacoma committed:
NEPT-2221: HTML tags in Webtools JSON Object field (#2347)

* NEPT-2221: Allow some tags when filtering html

* NEPT-2221: Update comment with the function used

* NEPT-2221: Add comment for the allowed html tags

enriquelacoma committed:
NEPT-2221: HTML tags in Webtools JSON Object field (#2347)

* NEPT-2221: Allow some tags when filtering html

* NEPT-2221: Update comment with the function used

* NEPT-2221: Add comment for the allowed html tags

enriquelacoma committed:
NEPT-446: Add Behat test for e-library feature (#2323)

enriquelacoma committed:
NEPT-446: Add Behat test for e-library feature (#2323)

aritomelo committed:
NEPT-2029: Unable to request translation "Data too long" error (#2307)

*   Unable to request translation "Data too long" error

  WIP: TEMPORARLLY PATCH applied, only for CODE REVIEW before it goes to dOrg.

  - Fixed the bug for body field

  - Implemented the logic to treat text_with_sumary and long_text field type, which origined the bug.

  - Implemented a logic for link test.

  - Break the whole logic into two function.

  - Small improvements and typo.

* WIP:Removed behat step just for testing.

* NEPT-2029: Re-insert the behat step to see if jenkins passed

* Nept-2029: Use the patch from dOrg.
  - Submit the patch on dorg, issue 2362321.
  - Update the make file to use the patch from dorg.
  - Removed the patch file from patches folder.

aritomelo committed:
NEPT-2029: Unable to request translation "Data too long" error (#2307)

*   Unable to request translation "Data too long" error

  WIP: TEMPORARLLY PATCH applied, only for CODE REVIEW before it goes to dOrg.

  - Fixed the bug for body field

  - Implemented the logic to treat text_with_sumary and long_text field type, which origined the bug.

  - Implemented a logic for link test.

  - Break the whole logic into two function.

  - Small improvements and typo.

* WIP:Removed behat step just for testing.

* NEPT-2029: Re-insert the behat step to see if jenkins passed

* Nept-2029: Use the patch from dOrg.
  - Submit the patch on dorg, issue 2362321.
  - Update the make file to use the patch from dorg.
  - Removed the patch file from patches folder.

enriquelacoma committed:
NEPT-2181: Display warning if node is scheduled for publishing at/after BREXIT (#2366)

* NEPT-2181: Change default message and date format

* NEPT-2181: Wrap text to avoid concat

* NEPT-2181: Fix coding

* Use date var

* NEPT-2181: Add missing vars

* NEPT-2181: Change default message and date format

AlbaParrilla committed:
Merge branch 'release-2.5.136' into nept-475

AlbaParrilla committed:
NEPT-475: Fix details in behat test for communities.

AlbaParrilla committed:
NEPT-475: Fix details in behat test for standard profile.

Fefaine committed:
Merge branch 'release-2.5' into release-2.5.136

AlbaParrilla committed:
Merge branch 'release-2.5.136' into nept-475

AlbaParrilla committed:
NEPT-475: Document multisite_registration feature.

AlbaParrilla committed:
NEPT-475: Add new scenarios in behat feature.

AlbaParrilla committed:
Merge branch 'nept-475' of https://github.com/ec-europa/platform-dev into nept-475

AlbaParrilla committed:
NEPT-475: Fix behat test.

AlbaParrilla committed:
NEPT-475: Fix mistake in readme file.

AlbaParrilla committed:
NEPT-475: Improve communities tests.

AlbaParrilla committed:
NEPT-475: Improve standard tests.

AlbaParrilla committed:
NEPT-475: Add theme_wip tag in communities.

Fefaine committed:
Merge branch 'release-2.5.137' into nept-475
